Source/FieldSolver/SpectralSolver/SpectralFieldData.cpp

```diff
@@ -227,6 +272,16 @@ SpectralFieldData::BackwardTransform( MultiFab& mf,
 #ifdef AMREX_USE_GPU
         // Add cuFFT-specific code ; make sure that this is done on the same
```

Comment:
Could you replace `Add cuFFT-specific code ` by `Perform FFT on GPU `?
Please keep the rest of the comment (`make sure that this is done on the same ...`), since it is helpful to someone who reads the code.

Reply:
Replaced the comment.

Reply:
Thank you @RemiLehe for the comments. I have incorporated them. Please let me know if you have more suggestions.